src/components/Calculator.jsx

```diff
@@ -0,0 +1,41 @@
+import CalculatorScreen from './CalculatorScreen';
+
+function Calculator() {
+  return (
+    <>
+      <div className="text-xl flex flex-col justify-center p-10">
+        <CalculatorScreen />
+        <div>
+          <div className="btn text-xl w-[5%] bg-slate-200">AC</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">+/-</div>
+          <div className="btn text-xl w-[5%] bg-slate-200">%</div>
+          <div className="btn text-xl w-[5%] bg-orange-400">/</div>
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you use the correct sign: &div; according to the design.